package-examples/wordpress/README.md

```diff
@@ -1,92 +1,85 @@
-wordpress
-==================================================
-
-# NAME
-
-  wordpress
-
-# SYNOPSIS
-  
-  This is an example of a kpt package which has a subpackage in it. 
-  Here are the steps to get, view, set and apply the package contents
-  
-  ### Fetch a remote package
-  Get the example package on to local using `kpt pkg get`
-  
-  $ kpt pkg get https://github.com/GoogleContainerTools/kpt.git/package-examples/subpackages/wordpress \
-    wordpress
-  
-      fetching package /package-examples/subpackages/wordpress from https://github.com/GoogleContainerTools/kpt to wordpress
-  
-  ### View the package contents
-  List the package contents in a tree structure.
-  
-  $ kpt cfg tree wordpress/
-  
-      wordpress
+# wordpress
+
+Here is an example to get, view, customize and apply contents of an example kpt package with a subpackage
+in its directory tree. As part of customization, we will be setting common namespace for both the packages
+and apply setters for individual packages.
+
+## Steps
+
+1. [Fetch the package](#fetch-the-package)
+2. [View the package contents](#view-the-package-contents)
+3. [Configure namespace](#configure-namespace)
+4. [Configure setter values](#configure-setter-values)
+5. [Render the declared values](#render-the-declared-values)
+6. [Apply the package](#apply-the-package)
+
+### Fetch the package
+
+Get the example package on to local using `kpt pkg get`
+
+    $ kpt pkg get https://github.com/GoogleContainerTools/kpt.git/package-examples/wordpress
```

Comment:
Rendering issue. Always use block quotes instead of indenting:

```shell
$ kpt
```

Can you also fix other example packages (copy-pasta here).

Reply:
Done.